src/deadline/job_attachments/upload.py

```diff
@@ -182,7 +179,7 @@ def upload_assets(
 
         if manifest_write_dir:
             self._write_local_manifest(
-                manifest_write_dir, manifest_name, full_manifest_key, manifest
+                manifest_write_dir, manifest_name, full_manifest_key, manifest, None
```

Comment:
What about adding default to the function parameter rather than passing `None` from the caller?